preprod.healdata.org/manifests/hatchery/hatchery.json

```diff
@@ -37,7 +37,7 @@
       "cpu-limit": "1.0",
       "memory-limit": "2Gi",
       "name": "(Generic) Jupyter Lab Notebook with R Kernel",
-      "image": "quay.io/cdis/jupyter-superslim-r:1.0.1",
+      "image": "quay.io/cdis/jupyter-superslim-r:markdown_preview",
```

Comment:
are we going to make a release of this?

Reply:
It's going to be picked up into the next release 1.0.3, who ever created that image didn't follow the naming convention for the release 1.0.2. BRH is currently on 'markdown_preview' and both HEAL and BRH will be moved to 1.0.3 in February or March.

Reply:
alright cool